gapis/resolve/state_tree.go

```diff
@@ -32,6 +32,15 @@ import (
 	"github.com/google/gapid/gapis/service/path"
 )
 
+const (
+	// If an array or slice has more than this many elements, it is transformed
+	// into multiple tree nodes, each with at most this many children.
+	// The threshold is applied twice, so if the array has more elements than this
+	// number *cubed*, the array root node will contain more than this many child
```

Comment:
> I think I understand what you're saying (that there's no point dealing with arrayGroupThreshold³ ?), but this is rather confusing.

Correct. I hope the new comment in the proto is not as confusing.

> Do you think the inline slice logic is worthwhile?

I see that it has only little value, but I think once we make the type part of the slice it can be a bit more useful. I think we should keep it.

> If you do think this is worthwhile, how do you feel about putting this variable in the path like you've done for the command tree?

Done.